setup.py

```diff
@@ -11,7 +11,7 @@
 optional_dependencies = {
     "docs": ["sphinx_mdolab_theme"],
     "test": ["testflo"],
-    "ffd": ["pygeo>=1.6.0"],
+    "ffd": ["pygeo>=1.15.0"],

```

Comment:
The latest version of pygeo is 1.14, and I don't think we should enforce the latest version unless there's a good reason